client/extensions/woocommerce/lib/order-values/test/index.js

```diff
@@ -116,3 +118,21 @@ describe( 'getOrderTotalTax', () => {
 		expect( getOrderTotalTax( orderWithCoupons ) ).to.eql( 4.4243 );
 	} );
 } );
+
+describe( 'getOrderRefundTotal', () => {
+	it( 'should be a function', () => {
+		expect( getOrderRefundTotal ).to.be.a( 'function' );
+	} );
+
+	it( 'should get the correct refund amount', () => {
+		expect( getOrderRefundTotal( orderWithCoupons ) ).to.eql( -10.0 );
+	} );
+
+	it( 'should return 0 if there are no refunds', () => {
+		expect( getOrderRefundTotal( orderWithoutTax ) ).to.eql( 0 );
+	} );
+
+	it( 'should get the correct tax amount with multiple refunds', () => {
```

Comment:
I think you meant "should get the correct refund amount with multiple refunds" here.